hnn_core/dipole.py

```diff
@@ -19,6 +20,10 @@ def simulate_dipole(net, n_trials=None, record_vsoma=False,
     net : Network object
         The Network object specifying how cells are
         connected.
+    tstop : float
+        The stop time (ms)
```

Comment:
This is different from your tstop description in `_drive_cell_event_times`. Can you change one of them to match?